config/uitests-amo.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  // For use on addons-server, or with the command `amo:ui-tests` and the
+  // HOSTNAME  variable as `ui-tests`.
```

Comment:
Instead of writing this, add a `NODE_CONFIG_ENV` entry in `package.json` with `ui-tests` as value.